workspaces/global-header/plugins/global-header/src/components/GlobalHeader.tsx

```diff
@@ -0,0 +1,377 @@
+    label: 'Create React App Template with annotations',
+  },
+  {
+    key: 5,
+    value: 'define-ansible-job-template',
+    label: 'Ansible Job Template',
+  },
+];
+
+interface MenuSectionProps {
+  children: React.ReactNode;
+  label: string;
+}
+
+const MenuSectionRoot = styled('li')`
+  list-style: none;
+
+  & > ul {
+    padding-left: 0;
+  }
+`;
+
+const MenuSectionLabel = styled('span')`
+  display: inline-block;
+  font-size: 0.875em;
+  alignment-baseline: baseline;
+  align-content: end;
+  color: #979797;
+`;
+
+function MenuSection({ children, label }: Readonly<MenuSectionProps>) {
+  return (
+    <MenuSectionRoot role="group">
+      <Box
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          margin: '0.75rem',
+        }}
+      >
+        <MenuSectionLabel>{label}</MenuSectionLabel>
+        <MenuItem>
+          <Link
+            underline="none"
+            to="/create"
+            style={{ display: 'inline-block', fontSize: '0.875em' }}
+          >
+            All templates
+          </Link>
+        </MenuItem>
+      </Box>
+      <ul style={{ marginLeft: 12, marginRight: 12 }}>{children}</ul>
+    </MenuSectionRoot>
+  );
+}
+
+export const GlobalHeader = () => {
+  const Search = styled('div')(({ theme }) => ({
+    position: 'relative',
+    borderRadius: `${theme.shape.borderRadius}`,
+    backgroundColor: 'transparent',
+    marginRight: theme.spacing(2),
+    marginLeft: 0,
+    width: '100%',
+    [theme.breakpoints.up('sm')]: {
+      marginLeft: 0,
+      width: 'auto',
+    },
+  }));
+
+  const SearchIconWrapper = styled('div')(({ theme }) => ({
+    padding: theme.spacing(0, 2),
+    height: '100%',
+    position: 'absolute',
+    pointerEvents: 'none',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  }));
+
+  const StyledInputBase = styled(InputBase)({
+    color: 'inherit',
+    width: '100%',
+    '& input': {
+      paddingLeft: '3rem',
+    },
+  });
+
+  const Listbox = styled('ul')(
+    ({ theme }) => `
+    font-family: 'IBM Plex Sans', sans-serif;
+    font-size: 0.875rem;
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    min-width: 200px;
+    border-radius: 3px;
+    overflow: auto;
+    outline: 1;
+    background: ${
+      theme.palette.mode === 'dark' ? theme.palette.background.default : '#fff'
+    };
+    border: 1px solid ${
+      theme.palette.mode === 'dark'
+        ? theme.palette.divider
+        : theme.palette.background.default
+    };
+    color: ${
+      theme.palette.mode === 'dark'
+        ? theme.palette.text.disabled
+        : theme.palette.text.primary
+    };
+    box-shadow: 0 4px 6px ${
+      theme.palette.mode === 'dark' ? 'rgba(0,0,0, 0.50)' : 'rgba(0,0,0, 0.05)'
+    };
+    z-index: 1;
+    `,
+  );
+  return (
+    <AppBar
+      position="sticky"
+      component="nav"
+      sx={{ backgroundColor: '#212427' }}
```

Comment:
Fine for now, but can we have a story for reevaluating all the colors and the theme/customization support for the header.

Reply:
Right, we will make this configurable too. Also, do you think it's possible to use colors from our theme? Mui `theme.palatte` only have a few fields available, and most of our default colors from theme plugin aren't.
https://mui.com/material-ui/customization/palette/#customization

Reply:
Let me address theme issues in a different story as it could be more complex.

Reply:
Created [RHIDP-5364](https://issues.redhat.com/browse/RHIDP-5364) to address the theme issue.